### DISCUSSION TITLE
Why is keptn itself not declerative?

### DISCUSSION THREAD

**Discussion by fractiunate** [keptn/keptn#7169] [Ideas]
I started a POC with keptn v0.13.1 and wondered why keptn itself is not usable declerative?

Wouldn't it be awesome (and logical?) to declare projects, services, quality gates etc. in a unified declerative way - is this listed on any roadmap?

Regards
David

**Comment by agardnerIT** [comment]
You'll be pleased to hear it's already being addressed and there's a WIP GitOps operator: https://github.com/keptn-sandbox/keptn-gitops-operator

https://github.com/keptn/enhancement-proposals/blob/kep/keptn_declarative_configuration/text/0069-declarative-configuration.md :)

**Reply by agardnerIT** [reply]
https://www.youtube.com/watch?v=AAvdz4aIayU
